Readme.md

```diff
@@ -74,7 +74,7 @@ What currently works:
 - [x] **TCP networking** (with this library)
 - [ ] **Filesystem access**
 - [x] **Environment variables**
-- [ ] **Multithreading**
+- [x] **Multithreading Executor**
```

Comment:
@jocaml Sto ti mislis?

Reply:
No, this is about guests having ability to spawn "threads"

Reply:
ok - then confusion was only mine.

closing